lib/tasks/dspace.rake

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "io/console"
+
+namespace :figgy do
```

Comment:
What's all this for? Is this just for testing, or is ingest a two step process - get the token and pass it in? If it's two steps, can we fold that in?